.templates/base_script.sh.template

```diff
@@ -0,0 +1,95 @@
+{% block header %}
+#!/usr/bin/env bash
+
+# Automatically generated by nengo-bones, do not edit this file directly
+
+NAME=$0
+COMMAND=$1
+STATUS=0  # used to exit with non-zero status if any command fails
+
+exe() {
+    echo "\$ $*";
+    args=( "$@" )
+    # remove empty spaces from args
+    for i in "${!args[@]}"; do
+      [ -n "${args[$i]}" ] || unset "args[$i]"
+    done
+    # put quotes around all arguments except the first
+    for i in "${!args[@]}"; do
+      if [[ $i -gt 0 ]]; then
+        args[$i]='"'${args[$i]}'"'
+      fi
+    done
+    eval "${args[@]}" || { echo -e "\033[1;31mCOMMAND '${args[0]}' FAILED\033[0m"; STATUS=1; }
```

Comment:
Looks like this part is the only change from the current base script, just for my own interest why was this required?

Reply:
> Are we expecting the nengo-bones PR to take a while or can we just wait for that to update this repo with SLURM?

There is no timeframe on that.. and seeing as everyone is busy, it might take a while.

> Looks like this part is the only change from the current base script, just for my own interest why was this required?

The `eval` is needed to properly process the aliases. In the `slurm-script`, it creates aliases for any `wrapped_commands` 
(e.g., `alias sphinx-build="srun -pCI sphinx-build"`), and while the aliases work in the bash terminal, I found that I needed to do `eval <command>` to get it to be recognized within the `exe` function.

Reply:
You should be able to just `extend` the base script instead of copying the whole thing actually, then just override the header block. to keep things cleaner (even though it's temporary).